src/pages/nft/claim/index.vue

```diff
@@ -525,34 +525,41 @@
           />
         </template>
         <template #footer>
-          <ButtonV2
-            v-if="shouldShowStartReadingButton"
-            class="phoneLg:w-full phoneLg:max-w-[480px]"
-            :content-class="['px-[48px]']"
-            :text="$t('nft_claim_claimed_button_start_reading')"
-            @click="handleStartReading"
-          />
-          <ButtonV2
-            v-if="claimingAddress"
-            :content-class="['px-[48px]']"
-            class="phoneLg:w-full phoneLg:max-w-[480px]"
-            :preset="isViewCollectionLoading ? 'plain' : 'tertiary'"
-            :text="
-              isViewCollectionLoading
-                ? $t('nft_claim_loading')
-                : $t('nft_claim_claimed_button_view_collection')
-            "
-            @click="handleViewCollection"
-          />
-          <ProgressIndicator v-else-if="isLoginLoading" />
-          <ButtonV2
-            v-else
-            :content-class="['px-[48px]']"
-            class="phoneLg:w-full phoneLg:max-w-[480px]"
-            preset="tertiary"
-            :text="$t('nft_claim_claimed_button_view_collection_sign_in')"
-            @click="handleClickSignIn"
-          />
+          <ProgressIndicator v-if="isViewCollectionLoading" />
+          <div v-else class="flex items-center w-full gap-[12px]">
+            <ButtonV2
+              v-if="shouldShowStartReadingButton"
+              class="phoneLg:w-full phoneLg:max-w-[480px]"
+              :content-class="['px-[48px]']"
+              :text="
+                isViewCollectionLoading
```

Comment:
Do we still need `isViewCollectionLoading` here if there is a big `v-if` in front?

Reply:
[🎨 Simplify button tex](https://github.com/likecoin/liker-land/pull/2057/commits/099f3f1fd9c05f951ff061e5e63cb180581a4476)